components/user/CourseCard.tsx

```diff
@@ -20,13 +19,16 @@ const CourseCard = ({ code }: { code: string }) => {
   const course = data.result
 
   return (
-    <Link href={`/courses/${code}`} className='flex flex-row gap-x-4 '>
+    <Link
+      href={`/courses/${code}`.toLowerCase()}
```

Comment:
can you change this to `/courses/${code.toLowerCase()}`, that localizes what needs to be lowercased

Reply:
Okay!